src/questions/dtos/test-case.dto.ts

```diff
@@ -0,0 +1,28 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsNumber, IsOptional } from 'class-validator';
+
+export class CreateTestCaseDto {
+  @ApiProperty()
+  @IsNotEmpty()
+  question_id: string;
```

Comment:
parameter not use snake case, just use camelCase as usually but when user input, use kebab in json